pkg/cmd/wheredoresourcescomefrom/areyoumymother.go

```diff
@@ -0,0 +1,23 @@
+package wheredoresourcescomefrom
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/reactionchain"
+	"github.com/spf13/cobra"
+)
+
+func NewResourceChainCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "where-do-resources-come-from",
+		Short: "ask your mother",
```

Comment:
lol

Reply:
@soltysh I wrote this entire pull just to write that sentence :)

Reply:
`wheredoresourcescomefrom` -> `ask your mother`